.github/workflows/main.yml

```diff
@@ -8,33 +8,39 @@ on:
     branches:
       - "**"
 
+env:
+  DOCKER_BUILDKIT: 1
+  URL_DOCKER_REGISTRY: ghcr.io
+  DOCKER_IMAGE_AGENCY: ghcr.io/absaoss/vcxagencynode/vcxagency-node:2.2.0
+  DOCKER_IMAGE_POOL: pstas/indypool-localhost:1.15.0-localhost
+
+  DOCKER_REPO_LOCAL_ALPINE_CORE: alpine-core
+  DOCKER_REPO_LOCAL_LIBVCX: libvcx
+  DOCKER_REPO_LOCAL_LIBVCX_TESTER: libvcx-tester
+  DOCKER_REPO_LOCAL_CODECOV: codecov
+  DOCKER_REPO_LOCAL_ANDROID: android-test
+
 jobs:
   workflow-setup:
```

Comment:
Can this step be extracted to a composite action?

Reply:
done